docs/reference/esql/functions/st_y.asciidoc

```diff
@@ -0,0 +1,33 @@
+[discrete]
+[[esql-st_y]]
+=== `ST_Y`
+
+*Syntax*
+
+[.text-center]
+image::esql/functions/signature/st_y.svg[Embedded,opts=inline]
+
+*Parameters*
+
+`point`::
+Expression of type `geo_point` or `cartesian_point`. If `null`, the function returns `null`.
+
+*Description*
+
+Special function extracts the `x` coordinate from the supplied point.
```

Comment:
Same here about the "special" wording.

Reply:
Also, I think this is copy-paste and you forgot to adapt it to `st_y` (for example, in this line you say "extracts the `x` coordinate).

Reply:
I thought I double-checked this, but clearly not! Fixed in 379d6cdd5577a5def64ae4fece1351bac7e64da3